modules/claims_api/lib/claims_api/vbms_sidekiq.rb

```diff
@@ -33,17 +47,24 @@ def rescue_vbms_file_number_not_found(power_of_attorney)
       log_message_to_sentry(self.class.name, :warning, body: error_message)
     end
 
-    def upload_to_vbms(power_of_attorney, path)
-      uploader = VBMSUploader.new(
-        filepath: path,
-        file_number: power_of_attorney.auth_headers['va_eauth_pnid'],
-        doc_type: '295'
-      )
-      upload_response = uploader.upload!
-      power_of_attorney.update(
-        status: ClaimsApi::PowerOfAttorney::UPLOADED,
-        vbms_new_document_version_ref_id: upload_response[:vbms_new_document_version_ref_id],
-        vbms_document_series_ref_id: upload_response[:vbms_document_series_ref_id]
+    private
+
+    def retrieve_veteran_file_number(power_of_attorney:)
+      ssn = power_of_attorney.auth_headers['va_eauth_pnid']
+
+      begin
```

Comment:
more general ruby question: isn't this `begin` not strictly needed and the rescue would still work as intended within the method block?

Reply:
Yeah. I usually write it like this and then byebug gripes at me.  But it seems like sometimes byebug doesn't enforce it.  🤷